go/client/cmd_chat_listchannels.go

```diff
@@ -70,9 +70,12 @@ func (c *CmdChatListChannels) Run() error {
 
 	ui.Printf("Listing channels on %s:\n\n", c.tlfName)
 	for _, c := range listRes.Convs {
-		convLine := fmt.Sprintf("#%s", c.Channel)
```

Comment:
fixes cases like:
```
$ keybase chat create-channel --topic-type dev crbot.node_test "%^"           
Creating a new PRIVATE DEV conversation [%^]
Success!
$ keybase chat list-channels --topic-type dev crbot.node_test                 
Listing channels on crbot.node_test:

#%!^(MISSING) (created by: dxb on: 2018-12-20)
```

Reply:
I'm surprised this fails like that. Formatting placeholders should be perfectly legal and fine in *printf vaargs. I made sure that's the case here: https://play.golang.org/p/kxNGSxNRqdH

The issue is when it's called like this: `fmt.Printf(userProvidedStr)` when you just want to print something from the user to the stdout. What am I missing?